rust/store/src/lib.rs

```diff
@@ -276,3 +276,22 @@ impl Store {
         })
     }
 }
+
+
+mod test {
+    use super::{
+        ToTypedValue,
+        TypedValue,
+        Timespec,
+    };
+
+    #[test]
+    fn test_timespec_to_typed_value() {
+        let timespec = Timespec {
+            sec: 1518434618,
+            nsec: 740993000,
```

Comment:
We have tests for truncation:

https://github.com/mozilla/mentat/commit/ebb77d59bc7aeab94f46efcf70759fbfe2d0830f#diff-b35d96f723d9e1ad4e01cbd9840b2754R737

It would be nice, here, to see a non-`000` `nsec`, so we can see that you're truncating, too.